setup.py

```diff
@@ -1,16 +1,21 @@
 import sys
+import warnings
 from distutils.sysconfig import get_config_var
 from os import environ
 from pathlib import Path
 from shutil import copy
 
+# safe to import numpy here thanks to pep518
+import numpy as np
 from setuptools import Command, Extension, setup
 
 # Default to using cython, but use the .c files if it doesn't exist
 try:
     from Cython.Build import cythonize
 except ImportError:
-    cythonize = False
+    wmsg = "Cython unavailable, unable to cythonize cf-units extensions!"
+    warnings.warn(wmsg)
+    cythonize = None
```

Comment:
```suggestion
    cythonize = False
```

Previous was clearer in IMO